review_checklist.rst

```diff
@@ -29,6 +16,32 @@ Social aspects:
   - If somebody suggests a good idea, mention it or put a thumbs up.
   - After merging, thank the author and reviewers for their time and effort.
 
+If you are a committer, also refer to the `Ansible committer guidelines <https://docs.ansible.com/ansible/devel/community/committer_guidelines.html>`_.
+
+Reviewing bug reports
+=====================
+
+When users report bugs, the first thing that is needed is to verify the behaviour.
+
+* [ ] Verify if the user made an obvious mistake in the example code. We often see user errors reported as bugs.
```

Comment:
```suggestion
* [ ] Verify user has made an obvious mistake in the example code. We often see user errors reported as bugs.
```

Reply:
@aminvakil I think the sentence is correct (i've just asked a native speaker). I committed the second comment, thank you!